csv-bulk-signature/src/main.ts

```diff
@@ -107,10 +109,10 @@ function generateSignatures(
         "for missing required fields (scroll up to see list)"
       );
     });
+  zipUpFile();
```

Comment:
moved this function call to outside of `.on` block. was resulting in EventEmitter errors due to piping multiple readstreams into the same zip stream